source/OptimaJet.DWKit.StarterApplication/Repositories/UserRepository.cs

```diff
@@ -26,46 +27,69 @@ IDbContextResolver contextResolver
             this.CurrentUserContext = currentUserContext;
         }
 
-        protected IQueryable<User> DefaultGet()
+        public override IQueryable<User> Filter(IQueryable<User> query, FilterQuery filterQuery)
```

Comment:
Please add an explanation of why `Get()` always return the `CurrentUser` if if the current user isn't in the specified organization (using the `Organization: orgId` header) and that the user of the api will still need to add a `filter[organization-id]` to keep from getting the CurrentUser in the results.